org.eclipse.debug.ui/ui/org/eclipse/debug/ui/CommonTab.java

```diff
@@ -702,19 +712,39 @@ protected void updateLaunchInBackground(ILaunchConfiguration configuration) {
 	 * @param configuration the local configuration
 	 */
 	private void updateEncoding(ILaunchConfiguration configuration) {
+		IPreferenceStore store = DebugUIPlugin.getDefault().getPreferenceStore();
+		boolean forceSystemEncoding = store.getBoolean(IDebugPreferenceConstants.CONSOLE_FORCE_SYSTEM_ENCODING);
+		forceSystemEncoding = getAttribute(configuration, DebugPlugin.ATTR_FORCE_SYSTEM_CONSOLE_ENCODING,
```

Comment:
What will be the functional difference between user selecting the Radio button as "Forced" and"Other UTF-8" ?

Reply:
The "forced" just doesn't specify **any** encoding, to the JVM decides which encoding to use (so it forces to JVM default encoding choice)

Reply:
If it is not a Java Launch configuration, like for C++ etc what will happen?